lxd/resource_lxd_cached_image.go

```diff
@@ -88,6 +94,10 @@ func resourceLxdCachedImageCreate(d *schema.ResourceData, meta interface{}) erro
 	if err != nil {
 		return err
 	}
+	if p, ok := d.GetOk("project"); ok && p != "" {
```

Comment:
nit: I recommend using `v, ok` which is more consistent with other coding patterns in this project. This comment applies to all occurrences of where you have:

```
if p, ok := ...
```